<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-09T11:45:50 IST" name="Default suite" finished-at="2023-06-09T11:46:00 IST" duration-ms="9733">
    <groups>
    </groups>
    <test started-at="2023-06-09T11:45:50 IST" name="Default test" finished-at="2023-06-09T11:46:00 IST" duration-ms="9733">
      <class name="api.test.PetTests">
        <test-method is-config="true" signature="setupData()[pri:0, instance:api.test.PetTests@1a18644]" started-at="2023-06-09T11:45:50 IST" name="setupData" finished-at="2023-06-09T11:45:51 IST" duration-ms="195" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupData -->
        <test-method signature="testAddPet()[pri:1, instance:api.test.PetTests@1a18644]" started-at="2023-06-09T11:45:51 IST" name="testAddPet" finished-at="2023-06-09T11:45:54 IST" duration-ms="3222" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddPet -->
        <test-method signature="testGetPet()[pri:2, instance:api.test.PetTests@1a18644]" started-at="2023-06-09T11:45:54 IST" name="testGetPet" finished-at="2023-06-09T11:45:55 IST" duration-ms="1567" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetPet -->
        <test-method signature="testUpdatePet()[pri:3, instance:api.test.PetTests@1a18644]" started-at="2023-06-09T11:45:55 IST" name="testUpdatePet" finished-at="2023-06-09T11:45:58 IST" duration-ms="2569" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [405]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at api.test.PetTests.testUpdatePet(PetTests.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdatePet -->
        <test-method signature="testDeletePet()[pri:4, instance:api.test.PetTests@1a18644]" started-at="2023-06-09T11:45:58 IST" name="testDeletePet" finished-at="2023-06-09T11:46:00 IST" duration-ms="2140" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeletePet -->
      </class> <!-- api.test.PetTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
